src/libs/template-parser.ts

```diff
@@ -335,6 +335,83 @@ const TemplateParserHelpers = function (request: Request) {
         }
       );
     },
+    // Get's the index of a search string within another string.
+    indexOf: function (
+      data: string | SafeString | HelperOptions,
+      search: string | SafeString | HelperOptions | undefined,
+      position?: number | string | SafeString | HelperOptions | undefined
+    ) {
+      data =
+        typeof data === 'object' && !(data instanceof SafeString)
+          ? ''
+          : data.toString();
+
+      search =
+        (typeof search === 'object' || typeof search === 'undefined') &&
+        !(search instanceof SafeString)
+          ? ''
+          : search.toString();
+
+      position =
+        (typeof position === 'object' || typeof position === 'undefined') &&
+        !(position instanceof SafeString)
+          ? undefined
+          : Number(position.toString());
+
+      if (typeof position === 'number') {
+        return data.toString().indexOf(search.toString(), position);
+      } else {
+        return data.toString().indexOf(search.toString());
+      }
+    },
+    // Returns if the provided search string is contained in the data string.
+    includes: function (
+      data: string | SafeString | HelperOptions,
+      search: string | SafeString | HelperOptions | undefined
+    ) {
+      data =
+        (typeof data === 'object' || typeof data == 'undefined') &&
+        !(data instanceof SafeString)
+          ? ''
+          : data.toString();
+
+      search =
+        (typeof search === 'object' || typeof search == 'undefined') &&
+        !(search instanceof SafeString)
+          ? ''
+          : search.toString();
+
+      return data.toString().includes(search.toString());
```

Comment:
`data` and `search` are already converted to string above
```suggestion
      return data.includes(search);
```